### ISSUE TITLE
generate_needy_initializer does not allow a default value of false

### ISSUE THREAD

**Issue by wezm** [luckyframework/lucky#978]
I tried to define a field:

```crystal
  needs required : Bool = false
```

but I get the error:

```
There was a problem expanding macro 'generate_needy_initializer'

Code in macro 'finished'

 1 | generate_needy_initializer
     ^
Called macro defined in lib/lucky/src/lucky/html_builder.cr:46:3

 46 | macro generate_needy_initializer

Which expanded to:

 > 15 |
 > 16 |
 > 17 |            @required : Bool,
                   ^
Error: argument must have a default value
```

It would appear that this line does not handle `false`:

https://github.com/luckyframework/lucky/blob/33606a60927df013f997f19dca3dc88439cf46e9/src/lucky/html_builder.cr#L52

**Comment by bdtomlin** [comment]
@jwoertink I have a few minutes this afternoon if you would like a PR for this. Just want to make sure no one else is working on it since it's a quickie:)

**Comment by jwoertink** [comment]
@bdtomlin Have at it! I'm definitely not working on a solution for this, so that'd be great.

**Comment by paulcsmith** [comment]
This has been on my todo list for a long time so your help would be greatly appreciated @bdtomlin :D

**Comment by wezm** [comment]
Fixed by #979
